### DISCUSSION TITLE
09C54ECC-3DBD-40F6-AC13-85D061490A8D

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#169] [General]
# Brown Dwarf Catalog

This catalog includes a grand total of 2819 ultra-cool dwarfs, spanning from the late M-type main sequence stars, to L, T and Y-type brown dwarfs. Over a hundred of them are actually binary systems, and three are triple, rendered here with approximate parameters.

https://celestia.mobi/addon?item=09C54ECC-3DBD-40F6-AC13-85D061490A8D&lang=it

**Comment by MirkoDifrancesco** [comment]
I cannot find any brown dwarfs...

**Reply by bird0cosmic** [reply]
Find known ones like Luhman-16
